lisk-dex-electron/src/Utils.js

```diff
@@ -24,4 +24,22 @@ const formatThousands = function (num, separator) {
 	return numParts.reverse().join(separator) + fractionDecimals;
 }
 
-export default formatThousands;
\ No newline at end of file
+const groupByKey = function (array, groupByKey, sumKey) {
+	return array.reduce((accumulator, newItem) => {
+        const groupByKeyValue = newItem[groupByKey];
+        if (groupByKeyValue in accumulator) {
+          const prevItem = accumulator[groupByKeyValue]
+          prevItem[sumKey] = prevItem[sumKey] + newItem[sumKey]
+          accumulator[groupByKeyValue] = prevItem;
+        } else {
+          accumulator[groupByKeyValue] = newItem;
+        }
+        return accumulator
+      }, {});
+}
+
+const Keys = (dict) => Object.keys(dict)
+
+const Values = (dict) => Object.values(dict)
```

Comment:
No big deal but I think these two wrappers `Values` and `Keys` are not necessary, it's OK to use `Object.values()` and `Object.keys()` in many parts of the code. It's not much to type out and saved having to import one more util.